src/main/cpp/subsystems/Elevator.cpp

```diff
@@ -41,26 +43,21 @@ void Elevator::SetElevator(double speed) {
   RightElevator.Set(ControlMode::MotionMagic, position);
 }*/
 
-/*void Elevator::SetUpTalons(){
-  RightElevator.ConfigSelectedFeedbackSensor(FeedbackDevice::CTRE_MagEncoder_Relative,
+void Elevator::SetUpTalons(){
+  mRightElevator.ConfigSelectedFeedbackSensor(FeedbackDevice::CTRE_MagEncoder_Relative,
 			kPID_PrimaryClosedLoop,
 			kTimeout_10Millis);
-	RightElevator.ConfigForwardSoftLimitEnable(false, kTimeout_10Millis);
-	RightElevator.ConfigReverseSoftLimitEnable(false, kTimeout_10Millis);
-	RightElevator.ConfigForwardLimitSwitchSource(LimitSwitchSource_FeedbackConnector, LimitSwitchNormal_NormallyOpen, kTimeout_10Millis);
-	RightElevator.ConfigReverseLimitSwitchSource(LimitSwitchSource_FeedbackConnector, LimitSwitchNormal_NormallyOpen, kTimeout_10Millis);
-	RightElevator.SetSensorPhase(true);
-	RightElevator.SetInverted(false);
-	RightElevator.ConfigPeakOutputForward(0.1, kTimeout_10Millis);
-	RightElevator.ConfigPeakOutputReverse(-0.02, kTimeout_10Millis);
-
-	LeftElevator.SetInverted(true);
-	LeftElevator.Follow(RightElevator);
+	mRightElevator.ConfigForwardSoftLimitEnable(false, kTimeout_10Millis);
+	//mRightElevator.ConfigForwardLimitSwitchSource(LimitSwitchSource_FeedbackConnector, LimitSwitchNormal_NormallyOpen, kTimeout_10Millis);
+	mRightElevator.ConfigReverseLimitSwitchSource(LimitSwitchSource_FeedbackConnector, LimitSwitchNormal_NormallyOpen, kTimeout_10Millis);
+	mRightElevator.SetSensorPhase(true);
+	mRightElevator.SetInverted(false);
+	mRightElevator.ConfigPeakOutputForward(0.1, kTimeout_10Millis);
+	mRightElevator.ConfigPeakOutputReverse(-0.02, kTimeout_10Millis);
 
-}*/
+	mLeftElevator.SetInverted(true);
```

Comment:
Change to `SetInverted(false)`

Reply:
the left side is supposed to be inverted; this line is good.